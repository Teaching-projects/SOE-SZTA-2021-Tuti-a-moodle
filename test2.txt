100
50
60
120
40
60