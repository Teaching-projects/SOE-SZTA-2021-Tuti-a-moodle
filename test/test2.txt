A
AL
100
50
60
10
B
BL
120
40
60
11
