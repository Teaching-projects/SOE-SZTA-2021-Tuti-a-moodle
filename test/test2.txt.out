Enter the name of the entity: Enter the lore of A: Enter the health of A: Enter the attack of A: Enter the defense of A: Enter the cooldown of A: 
Enter the name of the entity: Enter the lore of B: Enter the health of B: Enter the attack of B: Enter the defense of B: Enter the cooldown of B: 
====================
A - HP: 100.0, DMG: 50.0, DEF: 60.0
B - HP: 120.0, DMG: 40.0, DEF: 60.0
====================
There wouldn't be damage in combat!